DXMainClient/DXGUI/Multiplayer/ChatListBox.cs

```diff
@@ -28,24 +28,35 @@ private void ChatListBox_DoubleLeftClick(object sender, EventArgs e)
             if (SelectedIndex < 0 || SelectedIndex >= Items.Count)
                 return;
 
-            var link = Items[SelectedIndex].Text?.GetLink();
+            var link = Items[SelectedIndex].Text?.GetLink()?.ToLowerInvariant();
             if (link == null)
                 return;
 
-            bool regExpResult = _regexp.Match(link).Success;
+            bool result = false;
+
+            foreach (var elem in ClientConfiguration.Instance.AlwaysTrustedDomains)
+            {
+                if (string.IsNullOrEmpty(elem))
+                    continue;
+
+                result = result || link.Contains(elem, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
this will false positive because it will match `https://malicious.site/cncnet.org`